packages/apps/src/initSettings.ts

```diff
@@ -70,10 +75,21 @@ function getApiUrl (): string {
       : 'ws://127.0.0.1:9944'; // nothing found, go local
 }
 
+function migrateStoringAccountStorage () {
+  const localStorageVersion = store.get('localStorageVersion', 0) as unknown;
```

Comment:
I think you should've left the `|| 0` shortcircuit @Marcin-Radecki . Otherwise the logic is changed and the following `if` clause is never going to be true for people without `localStorageVersion` set. So to follow @youPickItUp 's advice, something like this would be in place:
```suggestion
  const localStorageVersion = store.get('localStorageVersion', 0) as unknown || 0;
```

Reply:
if I'm not wrong the `store.get('localStorageVersion', 0)` is enough, the second agrument is a fallback value

Reply:
Ahh, right! That solves it ✅